adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/MarkDuplicates.scala

```diff
@@ -96,14 +94,20 @@ private[rdd] object MarkDuplicates extends Serializable with Logging {
                           recordGroups: RecordGroupDictionary): RDD[SingleReadBucket] = {
     checkRecordGroups(recordGroups)
 
+    // Gets the library of a SingleReadBucket
+    def getLibrary(singleReadBucket: SingleReadBucket): Option[String] = {
```

Comment:
It might also be better to make `library` a method of `SingleReadBucket` to replace this `getLibrary` function.